packages/foreman/foreman/foreman.spec

```diff
@@ -267,21 +267,6 @@ Meta package to install requirements for OpenStack compute resource support.
 %files openstack
 %{_datadir}/%{name}/bundler.d/openstack.rb
 
-%package ovirt
```

Comment:
@theforeman/packaging, is this enough to remove the rpms from the host?
When users run `dnf update,` will it automatically remove all obsolete oVirt packages, or do I have to write some kind of migration to take care of this?

Reply:
No, you should add an `Obsoletes` statement that tells dnf to take care of it. Here's a good example:
https://github.com/theforeman/foreman-packaging/blob/8258c627534f90d72e7695f5b621f154a9ad50f1/packages/plugins/rubygem-foreman_google/rubygem-foreman_google.spec#L15-L16

I'm debating if it should be on the main foreman package or in foreman-obsolete-packages. Probably the former.